[Compute AG] adding all the azure AD OIDC based configuration missing steps and details for PCSUP-12002

priteshchandaliya opened the pull request.

priteshchandaliya committed:
adding all the azure AD OIDC based configuration missing steps and details.

Commit:
made some changes to doc and screenshot

Commit:
made some changes to doc and screenshot

rcaballeromx commented on compute/admin_guide/authentication/oidc.adoc:
```diff
@@ -164,7 +164,15 @@ You can only view the value of the secret when you create it. Copy and store the
+
+image::oidc_api_permission.png[width=250]
+
+. Add above group claim and select the *Groups assigned to the application* option.
```
@priteshchandaliya The steps above do not mention any groups, they are mentioned below. Do we need to add this step or should the step that follows needs to be modified?

Commit:
Fix markup, anonymize images, and copy-edit

Commit:
Add sub-steps for clarity, resize images

rcaballeromx commented:
@Pubs-MV The PR is ready for merging.

pfox1969 committed:
clarified when user-mapping is needed

added text that if they use the group-mapping method they should skip the user-mapping steps

pfox1969 committed:
grammar cleanup

pfox1969 reviewed (changes requested):
I don't think we need the image::oidc_client_secret.png in there. Because you can only copy the client secret's value at the time of creation. That screen cap is after the creation.

Commit:
Remove oidc_client_secret.png screenshot.

rcaballeromx reviewed (approved):
Thank you @pfox1969 for your changes.
I removed the screenshot as requested too.
@Pubs-MV I verified that the content builds.

Pubs-MV commented:
also related to PCSUP-10521
